docs/compilers/CSharp/Compiler Breaking Changes - DotNet 5.md

```diff
@@ -1,4 +1,4 @@
-﻿## This document lists known breaking changes in Roslyn in C# 9.0 which will be introduced with .NET 5 (Visual Studio 2019 version 16.8).
+﻿**This document lists known breaking changes in Roslyn in C# 9.0 which will be introduced with .NET 5 (Visual Studio 2019 version 16.8).**
```

Comment:
Please don't change these headers to bolds. Goes for the rest of the similar changes in this PR.

Reply:
In addition, the top-level header should be an H1, not an H2.

Reply:
I did this since I found the earlier variations of the document with regard to Visual Studio 2015. Attempting to normalize the format. Perhaps the older ones would be better with the H2 then in place of the bold?

Reply:
Yes, as these are titles, that's what they should be.